imports/ui/unit-explorer/unit-explorer.jsx

```diff
@@ -12,62 +12,169 @@ import { setDrawerState } from '../general-actions'
-                      <div className='ml3 mv1 semi-dark-gray lh-copy flex-grow overflow-hidden'>
-                        <div className='ti1 ellipsis'>{metaData.displayName || name}</div>
-                        <div className='flex'>
-                          <div className='ti1 ellipsis flex-grow'>{metaData.moreInfo || description}&nbsp;</div>
-                          <div
-                            onClick={evt => {
-                              evt.stopPropagation()
-                              dispatch(push(`/case/new?unit=${id}`))
-                            }}
-                            className='f6 ellipsis ml3 pl1 mv1 bondi-blue fw5 no-shrink'
-                          >
-                            Add case
-                          </div>
-                        </div>
-                      </div>
-                    </div>
-                  </MenuItem>
-                ))}
+        <RootAppBar title='My Units' onIconClick={() => dispatch(setDrawerState(true))} shadowless />
+        <div className='flex-grow flex flex-column overflow-hidden'>
+          <Tabs
+            className='no-shrink'
+            onChange={this.handleChange}
+            value={this.state.slideIndex}
+            inkBarStyle={{backgroundColor: 'white'}}
+          >
+            <Tab label='Active' value={0} />
+            <Tab label='Disabled' value={1} />
+          </Tabs>
+
+          <div className='flex-grow flex flex-column overflow-auto'>
+            <SwipeableViews
+              index={this.state.slideIndex}
+              onChangeIndex={this.handleChange}
+            >
+              {/* tab 1 */}
+              <div className='flex-grow bb b--very-light-gray bg-white'>
+                <FilteredUnitsList filteredUnits={activeUnits} dispatch={dispatch} currentUser={currentUser} />
+              </div>
+              {/* tab 2 */}
+              <div className='flex-grow bb b--very-light-gray bg-white'>
+                <FilteredUnitsList filteredUnits={disabledUnits} dispatch={dispatch} currentUser={currentUser} value={this.state.slideIndex} />
               </div>
-            </div>
-            <div className='absolute bottom-2 right-2'>
-              <FloatingActionButton
-                onClick={() => dispatch(push(`/unit/new`))}
-              >
-                <FontIcon className='material-icons'>add</FontIcon>
-              </FloatingActionButton>
-            </div>
+            </SwipeableViews>
           </div>
         </div>
+        <div className='absolute bottom-2 right-2'>
+          <FloatingActionButton
+            onClick={() => dispatch(push(`/unit/new`))}
+          >
+            <FontIcon className='material-icons'>add</FontIcon>
+          </FloatingActionButton>
+        </div>
       </div>
     )
   }
 }
 
+FilteredUnitsList.propTypes = {
+  filteredUnits: PropTypes.array,
+  currrentUser: PropTypes.string,
+  value: PropTypes.number
```

Comment:
"value" should be renamed as mentioned above, and it should be a boolean, not a number

Reply:
I am not sure why boolean value `showAddBtn={false}` with a reversed logic does not work as intended, while number ` showAddBtn={1}` does.  🤔🤔 The first one hides `Add Case` button in both active and disabled units, while second one hides it in disabled units. 

showAddBtn={true} raises StandardJS error  'value must be ommitted for boolean attributes`

Reply:
it sounds like `showAddBtn={false}` should have a reversed effect than `showAddBtn={1}`, so I'm not sure what the problem is. 
Using `showAddBtn={true}` is equivalent to just writing `showAddBtn` with nothing else after it, which is what StandardJS is complaining about.